File: Python/дз_4/1.py
# Вычислить число c заданной точностью d
# Пример:
# - при d = 0.001, π = 3.141.    10^(-1) ≤ d ≤10^(-10)

from math import pi

def inputNumbers(inputText):
    is_OK = False
    while not is_OK:
        try:
            number = int(input(f'{inputText}'))
            is_OK = True
        except ValueError:
            print('Введено нечисловое значение. Введите число.')
    return number

d =  inputNumbers("Введите число для заданной точности числа Пи:\n")
print(f'число Пи с заданной точностью {d} равно {round(pi, d)}')

File: Знакомство с вэб технологиями/дз_3/1.js
Math.round
let user_input = Number.parseFloat(prompt('Введите температуру в градусах Цельсия'));
alert(`Цельсия: ${user_input}\nФаренгейт: ${Math.round(((9 / 5) * user_input + 32)*10)/10}`)
